crates/frontend/src/components/function_form/function_form.rs

```diff
@@ -281,7 +283,17 @@ where
                     </div>
 
                     <div>
-                        <p class="text-red-500">{move || output_message.get()}</p>
+                        <p class="text-red-500">
+                            {move || {
+                                output_message
+                                    .get()
+                                    .map_or(
+                                        String::new(),
+                                        |o| { format!("{}\n{}", o.message, o.stdout) },
```

Comment:
Nice was doing it myself, just a request if you can do the same for the above error_message.get().

Reply:
Sure will do it